### PR TITLE
chore(concept): #578 Consistent null values

### PR DESCRIPTION
**Pull request by ds-crehm**
Created a concept with the aim to have "empty" values in a consistent format throughout Trace-X and show them in a user-friendliy and consistent way.

resolves eclipse-tractusx/traceability-foss#578

### CODE REVIEW

File: docs/concept/#578-consistent-null-values/null-value-display-empty.png
Comment (mkanal) [inline docs/concept/#578-consistent-null-values/null-value-display-empty.png:1]: Please remove that file.

File: docs/concept/#578-consistent-null-values/null-value-display-null.png
Comment (mkanal) [inline docs/concept/#578-consistent-null-values/null-value-display-null.png:1]: Please remove that file.

File: docs/concept/#578-consistent-null-values/null-value-display-placeholder.png
Comment (mkanal) [inline docs/concept/#578-consistent-null-values/null-value-display-placeholder.png:1]: Please remove that file.

**Review by mkanal — DISMISSED** [review]

File: docs/concept/#578-consistent-null-values/null-value-display-empty.png
Comment (ds-crehm) [reply docs/concept/#578-consistent-null-values/null-value-display-empty.png:1]: This one is still used

**Pull request merged**